db/migrate/20190610132813_add_user_id_to_post.rb

```diff
@@ -0,0 +1,24 @@
+class AddUserIdToPost < ActiveRecord::Migration[5.2]
+  def up
+    add_column :posts, :user_id, :integer
+
+    execute <<-SQL
+      INSERT INTO users (email, created_at, updated_at)
+        VALUES ('default@example.com', '2019-06-10 12:12:12', '2019-06-10 12:12:12')
+        RETURNING id;
+
+      UPDATE posts
+        SET user_id = id
+        WHERE user_id IS NULL;
+    SQL
```

Comment:
SQL! This migration is great. I'm curious what someone else things about the insert here. Might be good to do more of an upsert for safety? @dce maybe?

Reply:
I would be a little bit surprised if running migrations created a new user (especially in production). Given that you're just getting started on this project, I might just blow away the posts table (`execute "DELETE FROM posts").

Reply:
This is good stuff though, for sure -- good on ya for not invoking the model classes.